src/vue/common/TokenRequestDetails.vue

```diff
@@ -0,0 +1,335 @@
+<template>
+  <div class="token-request-details">
+    <div class="asset-details">
```

Comment:
again: token-request-details and asset-details. in the neighbor Vlad forcing naming "asset" instead of "token". For the consistency, we have to do the same thing here. Let’s rename all the `token`s into `asset`s (exception: TokenD, lol)

Reply:
Renamed